core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestion.java

```diff
@@ -68,11 +70,32 @@
 
     public static final int TYPE = 4;
 
+    private boolean skipDuplicates;
+
     public CompletionSuggestion() {
     }
 
-    public CompletionSuggestion(String name, int size) {
+    public CompletionSuggestion(String name, int size, boolean skipDuplicates) {
```

Comment:
I always get confused, is this a user-facing API? In that case, should we keep the original constructor with "false" as the default? If not all good.